aip/client-libraries/4231.md

```diff
@@ -219,6 +219,13 @@ to the following backwards-compatibility expectations:
   libraries.
 - The addition of a `google.api.resource_reference` annotation on an existing
   field **must** be a backwards-compatible change.
+- Changing a `google.api.resource_reference` from `child_type` to `type`
+  **must** be a backwards-compatible change when the `child_type` referenced
+  is single-parented and the newly referenced `type` is that exact parent.
+- For references that appear in a request, changing from `type` to
+  `child_type` **must** be a backwards-compatible change when the resource
+  originally referenced via the `type` is included in the possible resources
+  implied by the `child_type` resolution.
```

Comment:
Same question here, it would be great if we have an example,
```resource originally referenced via the `type` is included in the possible resources
  implied by the `child_type` resolution```
is not easy to understand.

Reply:
See other comment, this has been changed.